src/components/AddressSearch/index.js

```diff
@@ -21,7 +21,29 @@ import variables from '@styles/variables';
 import CONST from '@src/CONST';
 import CurrentLocationButton from './CurrentLocationButton';
 import isCurrentTargetInsideContainer from './isCurrentTargetInsideContainer';
-import isSearchedLocation from './isSearchedLocation';
+
+/**
+ * Check if it is the location by searching for the location
+ * @param {String} search the string to search for a location
+ * @param {Object} place the location data consisted of name, description and geometry
```

Comment:
NAB: `place the location data consisted of name, description and geometry` => `place The place to check for a match on the search`.

Also, please document each field of the place like so:
https://github.com/Expensify/App/blob/9adef6e47a56810e721b930566928dd08532710c/contributingGuides/STYLE.md#L165

Reply:
@cooldev900 Oh also please document place.name, place.description, etc.